package.json

```diff
@@ -49,10 +54,6 @@
   ],
   "license": "MIT",
   "main": "index.js",
-  "peerDependencies": {
-    "eslint": ">=6.2.2",
-    "eslint-plugin-react": ">=7.6.1"
-  },
```

Comment:
Please revert all version changes and keep this PR focused on adding support for `eslint-plugin-react-hooks` only.